package.json

```diff
@@ -8,7 +8,14 @@
     ],
     "scripts": {
         "install": "cd src/danfojs-base && yarn && cd ../danfojs-browser && yarn && cd ../danfojs-node && yarn",
-        "build": "cd src/danfojs-node && yarn build:clean && cd ../danfojs-browser && yarn build:clean",
-        "test": "cd src/danfojs-base && yarn && cd ../danfojs-node && yarn && yarn test:clean && cd ../danfojs-browser && yarn && yarn test:clean"
+        "build": "yarn build:browser && yarn build:node",
```

Comment:
This comes from my other PR #603 so I could run build and tests independently from the root of the project